src/pages/SettingsPage/UsersSettings/newServiceUser.jsx

```diff
@@ -0,0 +1,171 @@
+import { useRef, useState } from 'react'
+import { toast } from 'react-toastify'
+import styled from 'styled-components'
+import { SelectButton } from 'primereact/selectbutton'
+
+import { Button, SaveButton, Section, Dialog, FormRow } from '@ynput/ayon-react-components'
+import ayonClient from '@/ayon'
+import ApiKeyManager from '@/components/ApiKeyManager'
+import useUserMutations from '@/containers/Feed/hooks/useUserMutations'
+import { useAddUserMutation } from '@queries/user/updateUser'
+import copyToClipboard from '@/helpers/copyToClipboard'
+import callbackOnKeyDown from '@/helpers/callbackOnKeyDown'
+
+import UserAttribForm from './UserAttribForm'
+
+const FormRowStyled = styled(FormRow)`
+  .label {
+    min-width: 160px;
+  }
+`
+
+const NewServiceUser = ({ onHide, open, onSuccess }) => {
+  const {
+    password, setPassword,
+    formData, setFormData,
+    apiKey, setApiKey,
+    addedUsers,
+    resetFormData,
+  } = useUserMutations({})
+  const [addUser, { isLoading: isCreatingUser }] = useAddUserMutation()
+  const usernameRef = useRef()
+  const [keyName, setKeyName] = useState('');
+
+  const attributes = ayonClient.getAttribsByScope('user')
+
+  const validateFormData = (formData) => {
+    if (!formData.Username) {
+      return 'Login name must be provided';
+    }
+    if (apiKey == '') {
+      return 'Api key needs to be generated'
+    }
+
+    return null;
+  }
+
+  const preparePayload = (formData) => {
+    const payload = {
+      data: { isService: true },
+      name: formData.Username,
+    }
+
+    return payload;
+  }
+
+  const handleSubmit = async (close) => {
+    const validationResult = validateFormData(formData);
+    if (validationResult !== null) {
+      toast.error(validationResult)
+      return
+    }
+
+    try {
+      await addUser({ name: formData.Username, user: preparePayload(formData, attributes, password) }).unwrap()
```

Comment:
This needs to use `apiKey` instead of `password` as per docs: https://ayon.dev/docs#tag/Users/operation/create_user